runtime/moonbase/src/lib.rs

```diff
@@ -141,7 +143,9 @@ pub mod currency {
 }
 
 /// Maximum weight per block
-pub const MAXIMUM_BLOCK_WEIGHT: Weight = WEIGHT_PER_SECOND.saturating_div(2);
+pub const MAXIMUM_BLOCK_WEIGHT: Weight = WEIGHT_PER_SECOND
+	.saturating_div(2)
+	.set_proof_size(cumulus_primitives_core::relay_chain::v2::MAX_POV_SIZE as u64);
```

Comment:
:eyes:

Reply:
I set the same value as statemint here, but I dont think in any extrinsic we account for PoV size yet, so we should be fine